bitnami/fluentd/templates/service-accounts.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.forwarder.serviceAccount.create .Values.forwarder.enabled }}
```

Comment:
Another thing is that we use the suffix `serviceaccount` / `serviceaccounts`. For consistency with the other charts, could you also rename it?

Reply:
Could you clarify if you're talking about the template file name? 

If so, here I was just following the pattern of the `contour` chart which was suggested as an example and uses this name. But I agree consistency would be better, if we can find the pattern to follow.

However I do see that charts like `elasticsearch` have multiple service accounts in a template named `serviceaccount.yaml` (singular). There don't seem to be any with `serviceaccounts.yaml` (plural, no hyphen) as the name.

Reply:
In any case, I have reverted to the original name of `serviceaccount.yaml` as this seems the most consistent with the most number of charts.